app/views/general_shopping_lists/index.html.erb

```diff
@@ -0,0 +1,33 @@
+
+<div class="w-100 d-flex justify-content-center mt-4">
+<div class="w-75 d-flex flex-column">
+<h1 class="text-center">General Shopping Lists</h1>
+<div class="w-100 d-flex justify-content-between mt-4">
+<div>Amount of food items needed: <%= @recipe.ingredients.count%></div>
+<div>Total amount needed to buy: <span>$</span><%= @total_amount%></div>
+</div>
```

Comment:
Please stick to the wireframe provided where possible. Kindly add the `Recipe name` on the shopping list as per project requirements.
![image](https://user-images.githubusercontent.com/38088818/157822670-a7f6d991-4fb4-4366-b5a4-c88ee8648656.png)

Wireframe:
![image](https://user-images.githubusercontent.com/38088818/157822572-c8acb2b6-980f-4d21-83e7-37bc4cc2742b.png)